graphql_compiler/query_formatting/common.py

```diff
@@ -133,6 +146,7 @@ def deserialize_json_argument(
         GraphQLString: "Hello"
         GraphQLBoolean: True
         GraphQLID: "13d72846-1777-6c3a-5743-5d9ced3032ed"
+        GraphQLList: [1, 2, 3]
```

Comment:
I thought `GraphQLList` is not a subclass of `GraphQLScalarType`? If so, either the type hint for `expected_type` is incorrect, or this example is incorrect. Also, `GraphQLList` is an incomplete type, perhaps here you meant `GraphQLList(GraphQLInt)`?